common/components/WalletDecrypt/Trezor.tsx

```diff
@@ -101,6 +124,9 @@ export default class TrezorDecrypt extends Component<Props, State> {
             isLoading: false
           });
         } else {
+          if (res.error === 'TIMEOUT') {
```

Comment:
I'm not sure that res.error will ever be `TIMEOUT`. I think this is the exhaustive list of "errors": https://github.com/trezor/connect/blob/master/connect.js#L30

Between this and the comment above, I would probably just remove the showTip functionality on TREZOR and leave it at a loading indicator.

Reply:
Yea I don't actually have a TREZOR to test any of these changes on so I sort of just guessed it would be similar to ledger. I'll revert the TREZOR changes